src/validator.js

```diff
@@ -420,4 +429,19 @@ export default class Validator {
       return this.scan(_Xpi);
     }
   }
+
+  _getAddonArchitecture(xpiMetaData) {
+    // If we find a file named bootstrap.js this is assumed to be a
+    // Jetpack add-on: https://github.com/mozilla/amo-validator/blob/7a8011aba8bf8c665aef2b51eb26d0697b3e19c3/validator/testcases/jetpack.py#L154
```

Comment:
Nope. It's smart enough to realise it's a URL and not complain.

- tofumatt (Sent from mobile)

> On 13 Nov 2015, at 11:11, Stuart Colville <notifications@github.com> wrote:
> 
> In src/validator.js:
> 
> > @@ -420,4 +429,19 @@ export default class Validator {
> >        return this.scan(_Xpi);
> >      }
> >    }
> > +
> > +  _getAddonArchitecture(xpiMetaData) {
> > +    // If we find a file named bootstrap.js this is assumed to be a
> > +    // Jetpack add-on: https://github.com/mozilla/amo-validator/blob/7a8011aba8bf8c665aef2b51eb26d0697b3e19c3/validator/testcases/jetpack.py#L154
> Does eslint/jscs not whine about the length of this line?
> 
> —
> Reply to this email directly or view it on GitHub.
>